uint8[25] SAMPos12Avail
float64[25] SAMPosDegree
